sailthru/requirements/base.txt

```diff
@@ -1,2 +1,3 @@
+nose==1.3.7
```

Comment:
@rlucioni it was suggested by @clintonb here https://github.com/edx/ecommerce-scripts/pull/22#discussion_r87195920. So should I add requirements in a single or we should use nested requirements file?